avocado/core/nrunner.py

```diff
@@ -434,12 +434,13 @@ def run(self):
                                               [])
         for most_current_execution_state in super(ExecTestRunner, self).run():
             returncode = most_current_execution_state.get('returncode')
-            if returncode in skip_codes:
-                most_current_execution_state['result'] = 'skip'
-            elif returncode == 0:
-                most_current_execution_state['result'] = 'pass'
-            else:
-                most_current_execution_state['result'] = 'fail'
+            if returncode is not None:
+                if returncode in skip_codes:
+                    most_current_execution_state['result'] = 'skip'
+                elif returncode == 0:
+                    most_current_execution_state['result'] = 'pass'
+                else:
+                     most_current_execution_state['result'] = 'fail'
```

Comment:
lint is complaining about this, there is whitespace

Reply:
@richtja I've force pushed a fix for this whitespace.